globals/fields.go

```diff
@@ -0,0 +1,23 @@
+package globals
+
+const (
```

Comment:
We already use globals. This is just in a package called "globals" so that we acknowledge that fact. :-)

Reply:
Understood, but most of our previous globals were internal and were kept relatively close to where they were actually used (maybe not always... but hopefully more often than not).  Since these are exported values in a non-internal directory I would prefer better documentation on these fields.